src/services/goalServices/helpers.js

```diff
@@ -0,0 +1,22 @@
+const responsesForComparison = (g) => (g.responses || [])
```

Comment:
pulled this out from the recipient service so that we can use the same logic in two places